beta/kustomize/base/kustomization.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: op-scim
+resources:
```

Suggest we remove namespace from all Kustomizations. You can still deploy into a specific namespace by creating and switching context to that namespace or appending namespace as a parameter in the relevant `kubectl` command:

```suggestion
kind: Kustomization
resources:
```